src/Cli.Tests/InitTests.cs

```diff
@@ -164,8 +164,8 @@ bool expectSuccess
         /// </summary>
         [DataRow(null, "testcontainer", "", false, DisplayName = "Both database and schema are either null or empty.")]
         [DataRow("", "testcontainer", "testschema", false, DisplayName = "database is empty.")]
-        [DataRow("testDatabase", "testcontainer", "", false, DisplayName = "database is provided, Schema is null.")]
-        [DataRow("testDatabase", null, "", false, DisplayName = "database is provided, container and Schema is null/empty.")]
+        [DataRow("testDatabase", "testcontainer", "", true, DisplayName = "database is provided, Schema is null.")]
+        [DataRow("testDatabase", null, "", true, DisplayName = "database is provided, container and Schema is null/empty.")]
```

Comment:
What was the resolution to this thread?

Reply:
Actually test is correct, because schema is optional now. Previously it was mandatory information to pass during dab init.
I have updated the comment accordingly.